cmd/version.go

```diff
@@ -27,6 +34,18 @@ func versionCommand(l logger, host string, pluginRepo models.PluginRepository) *
 		RunE: func(c *cli.Command, args []string) error {
 			l.Printf(fmt.Sprintf("client: %s-%s", coloredNotice(config.Version), config.BuildCommit))
 
+			currentV, err := version.NewVersion(config.Version)
+			if err != nil {
+				return errors.Wrapf(err, "failed to parse current version")
+			}
+			latestV, err := version.NewVersion(checkLatestVersion())
+			if err != nil {
+				return errors.Wrapf(err, "failed to parse latest version")
+			}
+
+			if currentV.LessThan(latestV) {
+				l.Printf(fmt.Sprintf("new version is available : %s, consider updating the client", coloredNotice(latestV)))
+			}
```

Comment:
what if we move all this to `checkLatestVersion` and take `config.Version` as an argument? This will avoid failing for errors when parsing the latest version at line number 43.